gnomad_qc/v4/assessment/calculate_per_sample_stats.py

```diff
@@ -449,28 +511,16 @@ def main(args):
         raise ValueError("Stratifying by subset is only working on exomes data type.")
 
     try:
-        if args.create_per_sample_counts_ht:
-            chrom = "chr22" if test else None
-            if data_type == "exomes":
-                logger.info("Calculating per-sample variant statistics for exomes...")
-                mt = get_gnomad_v4_vds(
-                    test=test, release_only=True, split=True, chrom=chrom
-                ).variant_data
-            else:
-                logger.info("Calculating per-sample variant statistics for genomes...")
-                mt = get_gnomad_v4_genomes_vds(
-                    test=test, release_only=True, split=True, chrom=chrom
-                ).variant_data
-                ukb_capture_intervals = False
-                broad_capture_intervals = False
-
+        if args.create_filter_group_ht:
+            logger.info("Creating Table of filter groups for summary stats...")
             release_ht = release_sites(data_type=data_type).ht()
-            if test:
-                release_ht = hl.filter_intervals(
-                    release_ht, [hl.parse_locus_interval("chr22")]
-                )
+            filtering_groups_res = get_summary_stats_filtering_groups(
```

Comment:
I think it still makes more sense to get test filtering group HT too, by filtering to test-dataset and test-partitions, because the full filtering group Table will take very long to run, especially you're adding new annotations `variant_ac`, `variant_af`, it will be cost efficient to test and see if that's all we want.